.github/workflows/build-ci.yml

```diff
@@ -2,6 +2,7 @@ name: Node.js CI
 
 on:
   push:
+    branches: [master]
```

Comment:
I would change this to `branches-ignore: 'renovate/*'`

Reply:
I thought about it a bit more and when someone would create a or for another branch the ci build wouldn’t be triggered. 🤔

If we remove the branch filter from pr we have guaranteed builds for each pull request + we can limit push trigger to master only.

What is your idea behind ignoring only renovate branches?

Reply:
If someone makes a new branch (including on their own fork), they wouldn't get CI builds as they're pushing commits. The first time they would get a CI check is when they open a pull request. If there is a flaw early in the process, they wouldn't know until they were finished and ready to merge. CI is meant to give continuous feedback on the work you are doing.

The only reason that renovate is a problem is because it makes a branch and immediately opens a pull request triggering two simultaneous builds. I'd rather make an exception to the rules for something that has predictable behavior.

Reply:
Sometimes I need to push a commit to the renovate branches, e.g. adjusting code after breaking change in the API. So maybe we should instead disable the pr trigger for branches from renovate?

Reply:
Those would get checked in the PR because it reruns on new commits. 

Perhaps we should just leave it the way it is and deal with the long queue. The CI checks only take about 5 minutes. We're a public repository too, so we'll never go over a quota.

Reply:
The best solution would be that the `Node.js CI` action doesn't run for any commits that have already started a run. But, that's not possible at the moment.

Reply:
ok lets leave it like that tbh I just got annoying when trying to test the releases, usually, the impact shouldn't be that big.